website/addons/forward/tests/factories.py

```diff
@@ -12,6 +12,4 @@ class Meta:
         model = ForwardNodeSettings
 
     owner = SubFactory(ProjectFactory)
-    url = 'http://frozen.pizza.reviews/'
-    redirect_bool = True
-    redirect_secs = 15
+    url = 'http://frozen.pizza.reviews/'
```

Comment:
Yup. From http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206 :

> 3.206 Line
>
> A sequence of zero or more non- \<newline> characters plus a terminating \<newline> character.

This doesn't really matter for 99% of modern systems, but it's nice to be standards-compliant in the trivially-simple cases. Also, it can sometimes lead to odd terminal behavior:

```
~/D/example ❯ cat a.txt
foo% 
~/D/example ❯ cat b.txt
bar
~/D/example ❯ cat *.txt
foobar
```

`zsh` is smart enough to put a `%\n` after line two in this case, most shells are not.